omnimatter/prototypes/buildings/steam-omnitractor.lua

```diff
@@ -0,0 +1,88 @@
+local tractor_ingredients = {}
+local phlog_ingredients = {}
+if mods["angelsindustries"] and angelsmods.industries.components then
+	tractor_ingredients = {
+        {name="block-construction-1", amount=1},
+        {name="block-fluidbox-1", amount=3},
+        {name="block-omni-0", amount=3}
+    }
+    phlog_ingredients = {
+        {name="block-construction-1", amount=3},
+        {name="block-electronics-0", amount=2},
+        {name="block-fluidbox-1", amount=4},
+        {name="block-omni-0", amount=3}
+    }
+else
+	tractor_ingredients = {
```

Comment:
These 2 lines can be defined as the default, no need for the else (aka during the `local ....= {}` part